client/src/pages/private/ParticipantTable.js

```diff
@@ -412,10 +418,14 @@ export default () => {
           ];
 
         if (reducerState.tabValue === 'Hired Candidates') {
-          oldColumns = oldColumns.filter((column) => column.id !== 'engage');
+          // Remove existing engage, siteName and status column and force putting back siteName + status
+          oldColumns = oldColumns.filter(
+            (column) => column.id !== 'engage' && column.id !== 'siteName' && column.id !== 'status'
```

Comment:
```suggestion
            (column) => column.id !== 'engage' && column.id !== 'siteName' && column.id !== 'statusInfo'
```

Reply:
we need both status/statusInfo

Reply:
Gotcha, it's getting a bit verbose, can we do a
```
            !['engage', 'siteName', 'status', 'statusInfo'].includes(column.id)
```